src/main/java/org/roaringbitmap/RunContainer.java

```diff
@@ -2275,27 +2275,7 @@ private Container toEfficientContainer() {
     if (sizeAsRunContainer <= Math.min(sizeAsBitmapContainer, sizeAsArrayContainer)) {
       return this;
     }
-    if (card <= ArrayContainer.DEFAULT_MAX_SIZE) {
-      ArrayContainer answer = new ArrayContainer(card);
-      answer.cardinality = 0;
-      for (int rlepos = 0; rlepos < this.nbrruns; ++rlepos) {
-        int runStart = Util.toIntUnsigned(this.getValue(rlepos));
-        int runEnd = runStart + Util.toIntUnsigned(this.getLength(rlepos));
-
-        for (int runValue = runStart; runValue <= runEnd; ++runValue) {
```

Comment:
It's not possible to apply exactly same change for the Mapped version as there is a branch `BufferUtil.isBackedBySimpleArray` which is valid only for one case.